bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -87,6 +87,19 @@ Type of CO2 feedback.
 </entry>
 
 
+<entry id="o3_ppbv" type="real" category="clm_physics"
+       group="clm_inparm" valid_values="" >
+Atmospheric O3 molar ratio (by volume) only used when o3_type==constant (nmol/mol)
+(Set by CCSM_O3_PPBV)
+</entry>
+
+<entry id="o3_type" type="char*16" category="clm_physics"
+       group="clm_inparm" valid_values="constant,diagnostic" >
+Type of O3 feedback.
+    constant   = use the input o3_ppbv value
+    disgnostic = use diagnostic values from the atmosphere
+</entry>
+
```

Comment:
I see your point and consistency would be great, @billsacks. 
But I do not have any strong opinion on whether it should be "ozone" or "o3". Are there any guidelines regarding the usage of chemical formula vs colloquial names of species?

Reply:
At least for co2 and ch4, we use the chemical names. So I guess this implies using o3_method instead of ozone_method. I don't have strong feelings myself, though. As I write this out, though, I think I prefer something like o3_stress_method or o3_veg_stress_method rather than just o3_method: I'm realizing that o3_method is pretty vague. (Sorry, I think I might have suggested that name earlier.)

Reply:
Great comments about the naming convention! 

Would it make sense to do: 
```
ozone_metod --> o3_veg_stress_method 
o3_type --> o3_veg_stress_type 
o3_ppbv --> o3_veg_stress_ppbv 
```

Or does this get a little overkill? Personally, I also prefer to be precise rather than vague.

Reply:
The o3 forcing may be used for other applications besides just this veg stress parameterization. So o3_type and o3_ppbv probably shouldn't have "veg_stress" in their names. I could see renaming o3_type to o3_forcing_type. But it sounds like we're going to remove that variable from this PR, so we can come back to that naming.